src/mainview.cpp

```diff
@@ -37,23 +53,23 @@ CMainView::CMainView( QWidget *pParent ) :
 	}
 
 	QJsonObject JSONConfig = JSONConfigDocument.object();
-
 	for ( auto it = JSONConfig.begin(); it != JSONConfig.end(); it++ )
 	{
 		auto pHeader = new QLabel( it.key(), this );
 		pHeader->setObjectName( "Header" );
-		pLayout->addWidget( pHeader );
+		pLayout2->addWidget( pHeader );
 		QJsonArray arr = it.value().toArray();
-		for ( int i = 0; i < arr.size(); i++ )
+
+		for (auto && i : arr)
 		{
-			auto item = arr.at( i ).toObject();
+			auto item = i.toObject();
 
-			QPushButton *pButton = new QPushButton( this );
```

Comment:
Literally a useless change

Reply:
it's compliant with the rest of the code.

Reply:
Marking as resolved due to it complying with [comment](https://github.com/AWildErin/sdklauncher/pull/9#discussion_r1020842461)